changelog.d/9224.misc

```diff
@@ -0,0 +1 @@
+Add stable `/rooms/:roomId/aliases` endpoint from r0.6.0 (MSC2432 historically).
```

Comment:
I think this is a bugfix?

Reply:
it could also be a feature :P 

I'm not sure this case was considered when building the changelog system, so happy to do whatever people are most happy with.

Reply:
yeah sounds like it should be a feature. also:

```suggestion
Add new endpoint `/_matrix/client/r0/rooms/{roomId}/aliases` from Client-Server API r0.6.1 (previously [MSC2432](https://github.com/matrix-org/matrix-doc/pull/2432)).
```